### PR TITLE
ref(native): Add note about multi-process support

### PR DESCRIPTION
**Pull request by kahest**
Add note about multi-process support with crashpad using separate or shared database paths

### CODE REVIEW

File: src/platforms/native/configuration/backends/crashpad.mdx [hunk 1]
```diff
@@ -37,6 +37,11 @@ according to shell lookup rules at runtime.
 It is advised to specify both the path to the `crashpad_handler` executable
 as well as the path to the database directory manually.
 
+Multi-process use cases are also supported. Initialize Sentry individually
+in each process which spawns its own Crashpad handler either in a separate
+database path, or using a shared database path. Each run uses its individual
+run directory, and there is filesystem-level locking in place.
```
Comment (shanamatthews) [inline src/platforms/native/configuration/backends/crashpad.mdx:43]:
```suggestion
You can also use Crashpad for multi-process use cases. To do this, initialize Sentry individually
in each process that spawns its own Crashpad handler, either in a separate
database path or using a shared database path. Each run will use its individual
run directory, and there is filesystem-level locking in place.
```

Comment (shanamatthews) [reply src/platforms/native/configuration/backends/crashpad.mdx:43]: I found this original sentence to be a little confusing, so hopefully this reword makes it more clear and accurately reflects what you meant!

**Review by shanamatthews — APPROVED** [review]

File: src/platforms/native/configuration/backends/crashpad.mdx (hunk 1 above)
Comment (kahest) [reply src/platforms/native/configuration/backends/crashpad.mdx]: Thanks for the suggestion! I adapted it a tiny bit, wdyt?

**Review by Swatinem — APPROVED** [review]

File: src/platforms/native/configuration/backends/crashpad.mdx (hunk 1 above)
Comment (shanamatthews) [reply src/platforms/native/configuration/backends/crashpad.mdx]: This looks awesome, thanks for taking the time to make sure this is really clear!

**Pull request merged**